rust/main/Cargo.toml

```diff
@@ -287,6 +264,37 @@ features = ["aws"]
 git = "https://github.com/hyperlane-xyz/ethers-rs"
 tag = "2024-04-25"
 
+[patch.crates-io.bumpalo]
+git = "https://github.com/fitzgen/bumpalo"
+tag = "3.14.0"
+version = "=3.14.0"
```

Comment:
Since some of the dependencies were not locked compiling newer versions caused issues, not due to the upgrades but due to the dependencies of dependencies